ui/public/locales/en.json

```
@@ -3348,6 +3348,7 @@
 "message.reset.vpn.connection": "Please confirm that you want to reset VPN connection.",
 "message.linstor.resourcegroup.description": "Linstor resource group to use for primary storage.",
 "message.resize.volume.failed": "Failed to resize volume.",
+"message.resize.volumne.processing": "Volume resize is in progress",
```

```suggestion
"message.resize.volume.processing": "Volume resize is in progress",
```

Update it wherever applicable.